src/app/admin/pages/users/users.component.ts

```diff
@@ -33,9 +55,10 @@ export class UsersComponent implements OnInit {
     private route: ActivatedRoute,
     private toaster: ToasterService,
     public authService: AuthService,
-  ) { }
+  ) {
+  }
 
-  ngOnInit() {
+  ngAfterViewInit() {
```

Comment:
What was the reason behind moving to this lifecycle hook?

Reply:
I wanted the processor-intensive rendering to happen after the view was fully initialized instead of at the start of the process to avoid locking up the UI. It only really makes a difference in Chrome.